integr8ly/integreatly-operator
# MGDAPI-1575

# Issue link
https://issues.redhat.com/browse/MGDAPI-1575

# What
Disabled user-monitoring label on namespaces so unwanted Thanos alerts no longer show on cluster UI.

# Verification steps
1. Provision a cluster **without** these changes and run RHOAM with the following commands:
```
export INSTALLATION_TYPE=managed-api
make cluster/prepare/local
make code/run 
```
2. Verify that namespaces have the label `openshift.io/user-monitoring = true`
3. Stop the operator with `ctrl + c` and start it again with `make code/run`
4. Wait for it to do one full reconcile
5. Check that namespaces have the label `openshift.io/user-monitoring = false`
